docs/products/eigenda/core-concepts/payments.md

```diff
@@ -5,9 +5,13 @@ sidebar_position: 3
 
 # Payments
 
-The Payments system streamlines user interactions with the EigenDA, offering clear, flexible options for managing network bandwidth. With the payments update, EigenDA will support two flexible payment modalities:
+The Payments system streamlines user interactions with EigenDA, offering clear, flexible options for managing network
+bandwidth. EigenDA supports two flexible payment modalities:
 
-1. **On-demand Bandwidth**: Users can make pre-paid payments for occasional bandwidth usage without time limitations or throughput guarantees, and payments are recored per blob dispersal request. Charges are applied only when the request is successfully validated by the disperser server, providing flexibility for users with dynamic bandwidth requirements. 
+1. **On-demand Bandwidth**: Users can make pre-paid payments for occasional bandwidth usage without time limitations or
```

Comment:
nit: when I read "pre-paid payments" it reads like a reservation.
Not super sure what's the best way to rephrase, but feel like turning the sentence on its head and mentioning the "per-dispersal charges" first would make more sense (and then at the end mention that the actual accounting requires ahead of time ETH deposits)

Reply:
32c07b6c6c1875771c98095068a547b1dc11d09b

I reworded to hopefully be more clear. I decided not to talk about the payment mechanism: this is just the intro, and the details are explained below (the introduction about reservation payments _also_ doesn't get into specifics of the mechanism)